conversation_simulator/src/StateMachine.js

```diff
@@ -142,67 +142,74 @@ export default class {
     )
   }
 
-  submitSolution (solutionPhrase, noPoints, source) {
-    let normalizedPhrase = solutionPhrase.toLowerCase().trim()
+  normalizeSolution (solution) {
+    return solution.toLowerCase().trim()
+  }
 
-    // Find a phrase matching the submitted phrase if one exists
+  // Finds the matching solution in a state's possible solutions
+  matchSolution (solutionPhrase) {
     let solution
     for (let possibleSolution in this.currentState.Solutions) {
       if (this.currentState.Solutions.hasOwnProperty(possibleSolution)) {
-        let submittedWords = this.splitString(normalizedPhrase)
+        let submittedWords = this.splitString(solutionPhrase)
         let solutionWords = this.splitString(possibleSolution)
-        if (possibleSolution !== 'default' && this.checkSolution(submittedWords, solutionWords)) {
+        if (possibleSolution !== 'default' && this.compareSolution(submittedWords, solutionWords)) {
```

Comment:
lets create a constant called default and use it across the file instead of 'default'

Reply:
default is a reserved keyword in JavaScript so I named the constant 'defaultPath'.